### PR TITLE
new option added - Easy Directory Navigation

### PR DESCRIPTION
**Pull request by gautamsi**
type '\' or '/' in filter box to go to currently highlighted directory
todo: prevent entering text '\' or '/' - am not sure how to do this, tried for an hour and let it at this point.

### CODE REVIEW

**Comment by hmatsuda** [comment]
Thanks for PR @gautamsi .
I think `enter` or `ctrl-j` key to select directory are easy same as  `/`.  I can't understand `/`'s pros, please describe more details.

**Comment by gautamsi** [comment]
here was my situation. 
I had very large directory structure for my node.js project. I wanted to quickly go to a file which is deep in somewhere. As a dev it is easy for me (more like habit) to type folder name and then type '\' or '/' to change directory vs return key. The list view looks more like autocomplete on a terminal vs an Explorer/Finder like window.  it is about how i am used to change directory on terminal or powershell. 

I think changing this to Alt Navigation Key option may make more sense instead of easy directory navigation.

File: lib/file-explorer-view.coffee [hunk 1]
```diff
@@ -102,15 +107,25 @@ class FileExplorerView extends SelectListView
...
+      new Minimatch(ignores, matchBase: true, dot: true)
+
     for ignoredName in ignoredNames
       return true if ignoredName.match(fileName)
+
+  schedulePopulateList: ->
+    if atom.config.get("file-explorer.easyDirectoryNavigation") is true
+      keydown = @getFilterQuery()
+      keydown = keydown.substr(-1)
+      if keydown is "\\" or keydown is "\/"
```
Comment (hmatsuda) [inline lib/file-explorer-view.coffee]: I prefer to custom easy navigation keymap using configuration API like below.
```
# file-explorer.coffee#16
    easyDirectoryNavigationKey:
      type: 'string'
      default: '/'
```

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "file-explorer",
   "main": "./lib/file-explorer",
-  "version": "0.2.3",
```
Comment (hmatsuda) [inline package.json]: A version number is automatically updated by `apm publish` command. You don't need change this line.
https://atom.io/docs/v0.192.0/hacking-atom-package-word-count

**Pull request closed without merging**